core/src/main/java/apoc/convert/Json.java

```diff
@@ -81,10 +82,19 @@ private static Map<String, Object> mapWithOptionalProps(Map<String,Object> mapEn
     @Context
     public org.neo4j.graphdb.GraphDatabaseService db;
 
+
+    @Procedure("apoc.json.validate")
+    @Description("apoc.json.validate('{json}' [,'json-path' , 'path-options']) - to check if the json is correct (returning an empty result) or not")
+    public Stream<StringResult> validate(@Name("json") String json, @Name(value = "path",defaultValue = "$") String path, @Name(value = "pathOptions", defaultValue = "null") List<String> pathOptions) {
+        return ((List<String>) JsonUtil.parse(json, path, Object.class, pathOptions, false, true))
+                .stream()
+                .map(StringResult::new);
+    }
+
```

Comment:
There is a bit of a discussion in the team currently about adding new methods to APOC. Would you mind allowing us a few days to get back to you about this?